src/app/routes/onDemandTV/getInitialData/index.js

```diff
@@ -10,13 +11,21 @@ import getEpisodeAvailability, {
   getUrl,
 } from '#lib/utilities/episodeAvailability';
 
-export default async ({ path: pathname, pageType }) => {
+import processRecentEpisodes from '#app/routes/utils/processRecentEpisodes';
+
+const getRecentEpisodesToggle = pathOr({ enabled: false, value: 4 }, [
```

Comment:
Could `{ enabled: false, value: 4 }` be a variable named `DEFAULT_TOGGLE_VALUE`? Just so it's clearer